sbol_utilities/sep_054_extension.ttl

```diff
@@ -79,35 +79,29 @@ sep054:includeObjectReferences rdf:type owl:DatatypeProperty ;
               rdfs:label "includeObjectReferences" .
 
 
-sep054:Module rdf:type owl:Class ;
-        rdfs:comment  '''`Module` extends the `Collection` class with the following fields:
+sep054:Package rdf:type owl:Class ;
+        rdfs:comment  '''`Package` extends the `Collection` class with the following fields:
+                * `version` [0,1], string: The value of this property SHOULD use SemVer to indicate the version of this package. This property SHOULD be set if and only if a package is a root package.
+                * `conversion` [1], boolean: Indicates if this is a native package (false) or conversion package (true)
+                * `dissociated` [0,1], string: If not set, this is an atomic package. If set, indicates this package is a dissociated package of the type identified by the string (see practices below).
                 * `hasDependency` [0,n], URI: `Dependency` child objects
+                # `subPackage` [0,n], URI: Indicates the set of `Package` objects aggregated in the `Package`.
                 The `displayId` of a `Module` SHOULD be `module`, unless it is also a `package`.
-                Every `member` of a `Module` MUST have the same value for `hasNamespace` as the module itself.''' ;
-        rdfs:subClassOf sbol:Collection ,
-    [ rdf:type owl:Restriction ; owl:allValuesFrom sep054:Dependency ; owl:onProperty sep054:hasDependency ; ] .
-
-sep054:hasDependency rdf:type owl:ObjectProperty ;
-        rdfs:comment '''This property links to a description of a package dependency''' ;
-               rdfs:subPropertyOf sbol:directlyComprises ;
-              rdfs:domain sep054:Module ;
-              rdfs:range sep054:Dependency ;
-              rdfs:label "dependencies" .
-
-
-sep054:Package rdf:type owl:Class ;
-        rdfs:comment  '''A Package is a systematic aggregation of modules intended to be distributed and used as a
-        coherent whole. When stored in a hierarchical structure (e.g., files in directories), the relationship between
```

Comment:
I believe that the comment about hierarchical directory structure should be kept.

Reply:
I added the comment back in at the bottom of the rds comment (lines 95-97).